trace-utils/Cargo.toml

```
@@ -1,14 +1,18 @@
 [package]
 name = "datadog-trace-utils"
-authors = ["David Lee <david.lee@datadoghq.com>"]
 edition.workspace = true
 version.workspace = true
 rust-version.workspace = true
 license.workspace = true
+autobenches = false
 
 [lib]
 bench = false
 
+[[bench]]
+name = "main"
+harness = false
+
```

```suggestion
path = "benches/main.rs"

```

That allows not shipping the benchmarks in php and have cargo not complain.